<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>307</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Tahoma</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Prepis textu</string>
  </property>
  <widget class="QLineEdit" name="vstupne_cislo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vstupne_cislo_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vstupne cislo</string>
   </property>
  </widget>
  <widget class="QLabel" name="vysledok_rozlozenia_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vysledok rozlozenia cisla: </string>
   </property>
  </widget>
  <widget class="QLabel" name="vysledok_rozlozenia">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vyznam_cisla_v_numerologii_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vyznam cisla v numerologii:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vyznam_cisla_v_numerologii">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="spustit_PushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Spustit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
